src/wallet/sqlite.cpp

```diff
@@ -44,6 +48,9 @@ SQLiteDatabase::~SQLiteDatabase()
     Close();
     LOCK(cs_sqlite);
     g_file_paths.erase(m_file_path);
+    if (g_file_paths.empty()) {
+        sqlite3_shutdown();
```

Comment:
In commit "Initialize and Shutdown sqlite3 globals" (5c78c9094a0158e4d466b696a3bbbcbd35b51b15)

Return value isn't checked here. Would suggest at least logging an error so we know if something has gone wrong.

Reply:
Done